### PR TITLE
Not found

### PR DESCRIPTION
**Pull request by k0smo**

### CODE REVIEW

**Review by triciasykes — CHANGES_REQUESTED** [review]
Please put your real Not Found text in the heading. :)

**Review by triciasykes — CHANGES_REQUESTED** [review]
You still need to change the format of the Component ( export default...)

**Review by k0smo — COMMENTED** [review]
Export added to notfound

**Review by triciasykes — CHANGES_REQUESTED** [review]
Almost, but not quite.  There's no export on the test page.  look at this page in syllabus: https://github.com/learn-academy-2022-delta/Syllabus/blob/main/react/create-react-app.md and find the example of basic class component.  (The current way you have the component structured will worked, it's just not what was requested)

**Review by k0smo — COMMENTED** [review]
updated not found

**Comment by k0smo** [comment]
updated not found

File: app/javascript/components/pages/NotFound.test.js [hunk 1]
```diff
@@ -1,22 +1,19 @@
...
+    const foundHeading = notFound.find("h1")
+    const notFoundHeading = notFound.find("h3")     
+    expect(foundHeading.length).toEqual(1)
+    expect(notFoundHeading.text()).toEqual("Not Found")
   })
-})
\ No newline at end of file
+})
+export default NotFound
+
```
Comment (triciasykes) [inline app/javascript/components/pages/NotFound.test.js:18]: just delete line 17 - you don't need to export default in a test file

**Review by triciasykes — CHANGES_REQUESTED** [review]
See note below on the test.

**Review by k0smo — COMMENTED** [review]
tried again

**Review by k0smo — COMMENTED** [review]
deleted 17

File: app/javascript/components/pages/NotFound.test.js (hunk 1 above)
Comment (k0smo) [reply app/javascript/components/pages/NotFound.test.js:18]: bye bye line 17

**Review by k0smo — COMMENTED** [review]
NT found

**Review by k0smo — COMMENTED** [review]
not found

**Review by k0smo — COMMENTED** [review]
ntfd

**Review by triciasykes — APPROVED** [review]

**Pull request merged**